res/macos/run-space-acres.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+LAUNCH_DIR="$(pwd)"
+
+loaders=/tmp/saloaders.cache
+if [ $loaders ]; then
+  rm $loaders
+fi
+
+cp ../Frameworks/loaders.cache $loaders
+sed -i '' "s|PREFIX|$(pwd)/../Frameworks|g" $loaders
+
+export GDK_PIXBUF_MODULEDIR="$LAUNCH_DIR/../Frameworks/loaders"
+export GDK_PIXBUF_MODULE_FILE=$loaders
```

Comment:
I believe you shouldn't need to do any of this if you place the files relatively to each other the same way it is done on Windows. I don't have Mac to experiment with this and we can live with this for now, but it is quite awkward and really shouldn't be necessary.

You can see where all the files are stored in Windows CI.

Specifically I looked into dmg file and I think if you move `space-acres` from `MacOS` to `Frameworks/bin` and store loaders under `Frameworks/lib/gdk-pixbuf-2.0/2.10.0` it will "just work". It may require replacing path once during build and not doing this hack with moving it into `/tmp` that I don't understand the need for.